go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/grafana/grafana
 
-go 1.20
+go 1.21
```

Comment:
you should leave this unless you're using specific features from 1.21

Reply:
ya -- the hope is to use the 1.21 features :)    it moved a bunch of things from `golang.org/x/exp/...` to the standard import path

Reply:
nicee